cmake/compilers/MSVC.cmake

```diff
@@ -72,4 +72,11 @@ endif()
 set(TBB_IPO_COMPILE_FLAGS $<$<NOT:$<CONFIG:Debug>>:/GL>)
 set(TBB_IPO_LINK_FLAGS $<$<NOT:$<CONFIG:Debug>>:-LTCG> $<$<NOT:$<CONFIG:Debug>>:-INCREMENTAL:NO>)
 
-set(TBB_OPENMP_FLAG /openmp)
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
+    if (CMAKE_SYSTEM_PROCESSOR MATCHES "(x86|AMD64)")
+        set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -mrtm -mwaitpkg)
+    endif()
+    set(TBB_OPENMP_LINK_FLAG " ")
```

Comment:
Is it really needed? What if just skip setting this flag?

Hmm, but you'll likely have to put setting `LINK_FLAGS` property under `if (TBB_OPENMP_LINK_FLAG)`

Reply:
Never mind